src/grading-settings/grading-sidebar/index.jsx

```diff
@@ -1,39 +1,41 @@
 import React from 'react';
-import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { useIntl } from '@edx/frontend-platform/i18n';
 import PropTypes from 'prop-types';
 
 import { HelpSidebar } from '../../generic/help-sidebar';
 import messages from './messages';
 
-const GradingSidebar = ({ intl, courseId, proctoredExamSettingsUrl }) => (
-  <HelpSidebar
-    courseId={courseId}
-    showOtherSettings
-    proctoredExamSettingsUrl={proctoredExamSettingsUrl}
-  >
-    <h4 className="help-sidebar-about-title">
-      {intl.formatMessage(messages.gradingSidebarTitle)}
-    </h4>
-    <p className="help-sidebar-about-descriptions">
-      {intl.formatMessage(messages.gradingSidebarAbout1)}
-    </p>
-    <p className="help-sidebar-about-descriptions">
-      {intl.formatMessage(messages.gradingSidebarAbout2)}
-    </p>
-    <p className="help-sidebar-about-descriptions">
-      {intl.formatMessage(messages.gradingSidebarAbout3)}
-    </p>
-  </HelpSidebar>
-);
+const GradingSidebar = ({ courseId, proctoredExamSettingsUrl }) => {
+  const intl = useIntl();
+  return (
+    <HelpSidebar
+      courseId={courseId}
+      showOtherSettings
+      proctoredExamSettingsUrl={proctoredExamSettingsUrl}
+    >
+      <h4 className="help-sidebar-about-title">
+        {intl.formatMessage(messages.gradingSidebarTitle)}
```

Comment:
These could also be replaced with `<FormattedMessage />` components, and then we don't need the `intl` object / `useIntl` hook at all.